src/cljs/harja/views/urakka/kulut.cljs

```diff
@@ -146,26 +148,36 @@
                      (-> @tila/yleiset :urakka :loppupvm))}]) ;pvm/jalkeen? % (pvm/nyt) --- otetaan käyttöön "joskus"
 
 
+(defn- koontilaskun-kk-formatter
```

Comment:
Tää on varmasti ihan OK näin, mutta tuntuis että yleisesti kaikkien näkymien tulisi pyrkiä formatoimaan päivämäärät samaan tapaan, esim harja.pvm tai harja.fmt olemassaolevien funkkareiden avulla. Tämä siis lähinnä konseptuaalista pohdintaa, en itse ymmärrä miksi laskupuolella tarvii olla eri tavalla?